app/views/directives/lifecycle-hook.html

```diff
@@ -1,133 +1,56 @@
+    <dl class="dl-horizontal left">
+      <div class="is-item-description">
+        <dt>Command:</dt>
+        <dd>
+          <code class="probe-command">
+            <span ng-repeat="arg in hookParams.execNewPod.command">
+              <truncate-long-text content="arg" limit="80" newline-limit="1" expandable="false" use-word-boundary="false"></truncate-long-text>
+            </span>
+          </code>
+        </dd>
+        <div ng-if="hookParams.execNewPod.env">
+          <dt>Environment Variables:</dt>
+          <dd>
+            <div ng-repeat="env in hookParams.execNewPod.env">
+              {{env.name}}&#8201;&#8594;&#8201;{{env.value}}
             </div>
-          </div>
-
-          <div class="read-only-tag-image" ng-if="hookParams.tagImages.length > 1">
-            <p class="read-only-info" ng-if="$first">
-              <span class="pficon pficon-info" aria-hidden="true"></span>
-              More than one image tag is defined. To change image tags, use the YAML editor.
-            </p>
-            {{tagImage.containerName}}&nbsp;&rarr;&nbsp;{{tagImage.to.namespace || namespace}}/{{tagImage.to.name}}
-          </div>
+          </dd>
         </div>
-      </div>
-
-      <div class="form-group failure-policy">
-        <label class="required picker-label">Failure Policy</label>
-        <ui-select ng-model="hookParams.failurePolicy" search-enabled="false" ng-disabled="view.isDisabled">
-          <ui-select-match>{{$select.selected}}</ui-select-match>
-          <ui-select-choices repeat="failurePolicyTypes in lifecycleHookFailurePolicyTypes">
-            {{failurePolicyTypes}}
-          </ui-select-choices>
-        </ui-select>
-
-        <div ng-switch="hookParams.failurePolicy">
-          <div class="help-block" ng-switch-when="Retry">Retry the hook until it succeeds.</div>
-          <div class="help-block" ng-switch-when="Abort">Fail the deployment if the hook fails.</div>
-          <div class="help-block" ng-switch-when="Ignore">Ignore hook failures and continue the deployment.</div>
+        <div ng-if="hookParams.execNewPod.volumes">
+          <dt>Volumes:</dt>
+          <dd>
+            <div ng-repeat="volume in hookParams.execNewPod.volumes">
+              {{volume}}
+            </div>
+          </dd>
         </div>
       </div>
-    </fieldset>
+    </dl>
   </div>
-
-  <span>
-    <a href="" role="button" ng-if="!hookParams" ng-click="addHook()">Add {{type}} lifecycle hook</a>
-    <a href="" role="button" ng-if="hookParams" ng-click="removeHook()">Remove {{type}} lifecycle hook</a>
-  </span>
-</ng-form>
+  <div ng-if="hookParams.tagImages">
+    <div ng-repeat="tagImage in hookParams.tagImages">
+    <h5 class="container-name">Container {{tagImage.containerName}}</h5>
+      <dl class="dl-horizontal left">
+        <div class="is-item-description">
+          <dt>Tag As:</dt>
+          <dd>
+            <a ng-href="{{tagImage.to.name | navigateResourceURL : 'ImageStreamTag' : namespace}}">
```

Comment:
Do you need to check first for `tagImage.to.namespace`? Then fall back `namespace` if not defined. (If `tagImage.to.namespace && tagImage.to.namespace !== namespace`, it should not be a link.)